index.js

```diff
@@ -27,7 +27,11 @@ const implementations = {
   },
   yarn: {
     install: (packages, isDev) => `yarn add ${isDev ? "-D" : ""} ${packages}`,
-    sync: "yarn",
+    sync: "yarn install --frozen-lockfile",
```

Comment:
`--frozen-lockfile` is deprecated in current versions of yarn (at least as of v3.5.0). It'll still work, it just gives a warning.

Reply:
haha, I'm on version 1.22 still 😬 .  I guess we could check `yarn --version` and use it accordingly